hera_pspec/pspecdata.py

```diff
@@ -335,6 +338,7 @@ def clear_cache(self, keys=None):
         if keys is None:
             self._C, self._I, self._iC, self._Y, self._R = {}, {}, {}, {}, {}
             self._identity_G, self._identity_H, self._identity_Y = {}, {}, {}
+            self._RegFactor = {}
```

Comment:
I don't see where `RegFactor` is used elsewhere in the code. Let's get rid of this if its not being used

Reply:
I've removed all mentions of RegFactor.